src/components/Checkbox/Checkbox.tsx

```diff
@@ -7,18 +7,27 @@ interface ICheckbox {
   value?: string;
   checked?: boolean;
   name?: string;
-  onChange?: (value: any) => void;
+  onChange: (value: any) => void;
   children?: React.ReactNode;
-  className?: string;
+  className: string;
+  size: 'small' | 'large';
 }
```

Comment:
we could move interfaces to a  type file on this folder

Reply:
Yes, and probably the "size property" should be a different prop because we are repeating the same thing almost always.